react-notification-system-redux/index.d.ts

```diff
@@ -0,0 +1,39 @@
+// Type definitions for react-notification-system-redux 1.0
+// Project: https://github.com/gor181/react-notification-system-redux
+// Definitions by: Karol Janyst <https://github.com/LKay>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Component } from "react";
+import { Action } from "redux";
+import { Attributes, Notification } from "react-notification-system";
+
+export as namespace Notifications;
+
+export = Notifications;
+
+declare class Notifications extends Component<Notifications.NotificationsProps, any> {}
+
+declare namespace Notifications {
+
+    export type NotificationsState = Notification[];
```

Comment:
i am getting this error when i setting reducer from "react-notification-system-redux" in combine reducers:
```
Type 'NotificationsReducer<any>' is not assignable to type 'Reducer<Notification[], AnyAction>'.
  Types of parameters 'state' and 'state' are incompatible.
    Type 'Notification[] | undefined' is not assignable to type 'Notification[]'.
      Type 'undefined' is not assignable to type 'Notification[]'.
```

in
`type NotificationsState = Notification[];` 
if i add any character inside [] it works any fix to this?